platform/kube/controller.go

```diff
@@ -568,6 +569,49 @@ func (c *Controller) HostInstances(addrs map[string]bool) []*model.ServiceInstan
 	return out
 }
 
+const (
+	IstioServiceAccountPrefix = "istio:"
+	LocalDomain = "local"
```

Comment:
Can you update `ServiceSuffix` so we use same domain suffix for services and service accounts?